.circle/vars.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 DISTROS=(wheezy jessie trusty centos7)
+TESTING=(wheezy jessie trusty)
 DISTRO=${DISTROS[$CIRCLE_NODE_INDEX]}
```

Comment:
Because we need packages for the required set. But don't need tests for specific distros... details here #31